mypy_django_plugin/main.py

```diff
@@ -60,6 +62,30 @@ def add_new_manager_base_hook(ctx: ClassDefContext) -> None:
     helpers.add_new_manager_base(ctx.api, ctx.cls.fullname)
 
 
+def reparametrize_manager_base_hook(ctx: ClassDefContext) -> None:
+    manager = ctx.api.lookup_fully_qualified_or_none(ctx.cls.fullname)
+    if manager is None or manager.node is None:
+        return
+    assert isinstance(manager.node, TypeInfo)
+
+    should_reparametrize = (
+        manager.node.bases
+        and manager.node.bases[0].args
```

Comment:
What if I position a different type as first base?

```python
from django.db.models.manager import BaseManager

class MySuperMixin:
    ...

class MyManager(MySuperMixin, BaseManager):
    ...
```

And since we've already "filtered" that the class we get inherits `BaseManager` in `get_customize_class_mro_hook`, we should be able to traverse all bases and assert that we find it.

Reply:
Good catch! Will fix now